api/server/init.go

```diff
@@ -56,6 +58,20 @@ func getEnvInt(key string, fallback int) int {
 	return fallback
 }
 
+func getEnvDuration(key string, fallback time.Duration) time.Duration {
+	var err error
+	res := fallback
+	if tmp := os.Getenv(key); tmp != "" {
+		res, err = time.ParseDuration(tmp + "s")
```

Comment:
Do we need to append "s"? If we expect a duration string, then we can just try to parse it?

Reply:
I added that because it would be nice going forward to be able to automatically compute the write time in helm templates using the read times and placement times. To be able to do arithmetic in helm, I removed the "s" at the from there. So we don't expect duration strings here, just the integral part in seconds.

Reply:
I see. In that case, may I suggest that we use parse int? We can get rid of this function and just use getEnvInt? (or add GetEnvUint?)

Reply:
or rename this as getEnvSeconds()?